### ISSUE TITLE
signal quality & battery values

### ISSUE THREAD

**Issue by heavyk** [openyou/emokit#34]
I have two machines. both are mac and both are lion. on the first, I'm using the standard driver. when I plug the headset into that machine, I can see the current battery charge level, but using emokit on this machine, it always says 0.

second, I don't seem to get signal quality either for any of the leads.

however, I think this problem could be bigger, because the gyros don't give me constant numbers... and I'm not moving my head. in the epoc app on the other machine, it works as it should.

am I doing something wrong? perhaps I'm not decoding it right? I'm just outputting all of the terminals with the recompiled bin/emokitd

I'd really like to get this working... it'd be awesome to get my own signals :)

**Comment by ahmedfarazkhan** [comment]
Hey, @heavyk did you manage to get nonzero signal quality values?

**Comment by heavyk** [comment]
at one point in time I did get all of the electrodes generating signal just fine, but since the headset is so awkward, I couldn't actually get all the electrodes to give me signal on my head (even though individually they did all work). I remember I had to wrap something around my head to keep the electrodes tight and then later I tried using a pin like conductors. painful to use, so I haven't really used it since then. I'm not even sure I know where it is now.
